src/node_jsvmapi.cc

```diff
@@ -23,6 +23,24 @@
 #include <vector>
 #include <string.h>
 
+static v8::Persistent<v8::Value> *lastException = 0;
```

Comment:
(You could also make it a private static member of the helper class I suggested in the other comment.)